terratest/test/k8gb_abstract_full_roundrobin_test.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2021 The k8gb Contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Generated by GoLic, for more details see: https://github.com/AbsaOSS/golic
+*/
+package test
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/require"
+	"k8gbterratest/utils"
+	"testing"
+)
+
+func AbstractTestFullRoundRobin(t *testing.T, n int) {
```

Comment:
Do we need to export this function? Isn't it the same test package where it is accessed?

Reply:
right, changed to lower case